res/map/AWBWMapConverter.py

```diff
@@ -141,6 +141,8 @@ def countryCodeToPlayerID(x):
 		'pl': 13,
 		'ar': 14,
 		'wn': 15,
+		'aa': 16,
+		'ne': 17,
```

Comment:
No SC?

Reply:
I think it's newer than my changes here, lol

Reply:
I didn't realize they were still adding new ones, though I guess AA and NE are also relatively new. SC does look pretty cool, though the tanks should probably be like stegosaurs or something (blah readability blah OK fine).

Reply:
That reminds me. I should steal the official sprites at some point, too.
I'm thinking I wait until they get the full intended 4.

Reply:
There's one more new one planned after AA, NE, and SC? Cool.